packages/webpack/mixin.core.js

```diff
@@ -219,6 +232,8 @@ WebpackMixin.strategies = {
   configureBuild: pipe,
   inspectBuild: sequence,
   getBuildConfig: callableSync,
```

Comment:
These strategy definitions need to be rename to `getWebpack{Build,Develop,Node}Config`, respectively.